neardev/dev-account

```diff
@@ -1 +1 @@
-dev-1654372257870-91029802957114
```

Comment:
you should probably be ignoring the `neardev` folder? It will just be re-created if it doesn't exist, unless you want people using this to point to the same account for some reason?